### PR TITLE
tickets/PREOPS-3384

### PR DESCRIPTION
**Pull request by MelissaGraham**
clarify the "how to become a delegate" process

### CODE REVIEW

File: dp0-delegate-resources/index.rst [hunk 1]
```diff
@@ -22,37 +22,61 @@ DP0 Delegate Homepage
...
+
+The number of delegates is currently limited to 600, but space is available.
 
 
 .. _Delegate-Homepage-Getting-Started-Checklist:
 
 Getting started with DP0 checklist
 ==================================
 
+Becoming a delegate is as easy as following this checklist and getting an account in the Rubin Science Platform (RSP).
```
Comment (GloriaFA) [inline dp0-delegate-resources/index.rst:41]: "Becoming a delegate and accessing the DP0"

File: index.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ Data Preview 0 (DP0) is the first of three data previews during the period leadi
 The goals of DP0 are to serve as an early integration test of the Legacy Survey of Space and Time (LSST) Science Pipelines and the Rubin Science Platform (RSP), and to enable a limited number of astronomers and students to begin early preparations for science with the LSST.
 
 .. Important::
-    **New to DP0?** Welcome! See the :ref:`Delegate-Homepage-Getting-Started-Checklist`.
+    To access DP0 and become a delegate, see the :ref:`Delegate-Homepage-Getting-Started-Checklist`.
```
Comment (GloriaFA) [inline index.rst:31]: "To become a delegate and access the DP0"

Comment (MelissaGraham) [reply index.rst]: incorporated -- that is better wording thank you!

File: dp0-delegate-resources/index.rst (hunk 1 above)
Comment (MelissaGraham) [reply dp0-delegate-resources/index.rst]: incorporated this wording too

**Review by GloriaFA — APPROVED** [review]

**Pull request merged**